iterator.go

```diff
@@ -559,11 +560,11 @@ func (it *Iterator) Close() {
 func (it *Iterator) Next() {
 	// Reuse current item
 	it.item.wg.Wait() // Just cleaner to wait before pushing to avoid doing ref counting.
+	it.scanned += len(it.item.key) + len(it.item.val) + len(it.item.vptr) + 2
```

Comment:
The general flow is we do `itr.Rewind(); itr.Valid(); itr.Next()`. so the item `it.item` is holding is valid, so we don't need `nil` check here.

+2 is for `meta` and `usermeta`.